src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/AbstractItemFileWatchDataSource.cs

```diff
@@ -39,10 +39,11 @@ protected override IDisposable LinkExternalInput(ITargetBlock<IProjectVersionedV
             IProjectValueDataSource<IProjectSubscriptionUpdate> source = _projectSubscriptionService.ProjectRuleSource;
 
             // Transform the changes from evaluation -> FileWatchData
-            DisposableValue<ISourceBlock<IProjectVersionedValue<FileWatchData>>> transformBlock = source.SourceBlock
-                                                                                                        .TransformWithNoDelta(update => update.Derive(u => CreateFileWatch(u.CurrentState)),
-                                                                                                            suppressVersionOnlyUpdates: true,
-                                                                                                            ruleNames: new[] { ItemSchemaName });
+            DisposableValue<ISourceBlock<IProjectVersionedValue<FileWatchData>>> transformBlock
+                = source.SourceBlock.TransformWithNoDelta(
```

Comment:
What changed here?

Reply:
Just formatting (f8a93d3ad218bf676c57defb266471b34b9c49da). I meant to pull this out to a different PR.